test/integration/proposals_config.ts

```diff
@@ -19,66 +16,8 @@ const proposals: ProposalsConfigMap = {
       category: ProposalCategory.DAO
     }
     */
-  fip_82: {
```

Comment:
All of these have executed and are majorly slowing down the e2e tests (10+ mins in total run time), have removed